packages/beacon-node/src/network/subnets/dllAttnetsService.ts

```diff
@@ -24,6 +24,9 @@ enum SubnetSource {
   longLived = "long_lived",
 }
 
+/** As monitored on goerli, we only need to subscribe 2 slots before aggregator dutied slot to get stable mesh peers */
+const SLOTS_TO_SUBSCRIBE_IN_ADVANCE = 2;
```

Comment:
@tuyennhv can you make this a hidden CLI option in another PR?

Reply:
@dapplion https://github.com/ChainSafe/lodestar/pull/5819